pkg/client/buildconfigs.go

```diff
@@ -31,6 +42,7 @@ type BuildConfigInterface interface {
 
 	Instantiate(request *buildapi.BuildRequest) (result *buildapi.Build, err error)
 	InstantiateBinary(request *buildapi.BinaryBuildRequestOptions, r io.Reader) (result *buildapi.Build, err error)
+	InstantiatePipeline(c *buildapi.JenkinsPipelineBuildStrategy, k kclient.Interface) error
```

Comment:
i'm not convinced this should be hung off the BuildConfig.  Creating the jenkins service is more a side-effect of the build config being created, than something being created from the buildconfig (which is what instantiate and instantiatebinary are doing)